geotrek/common/mixins/models.py

```diff
@@ -317,6 +319,13 @@ class PublishableMixin(BasePublishableMixin):
     class Meta:
         abstract = True
 
+    def save(self, *args, **kwargs):
+        if (self.pk and self.__class__.objects.get(pk=self.pk).review != self.review and self.review) and settings.ALERT_REVIEW:
+            subject = _("{obj} need a review").format(obj=self)
+            message = render_to_string('common/review_email_message.txt', {"obj": self})
+            mail_managers(subject, message, fail_silently=False)
```

Comment:
fail_silently=False -> unable to modify review attribute if mail server or configuration has problem